python/ray/data/tests/test_actor_pool_map_operator.py

```diff
@@ -126,6 +128,47 @@ def test_pending_to_running(self):
         assert pool.num_idle_actors() == 0
         assert pool.num_free_slots() == 3
 
+    def test_restarting_to_alive(self):
```

Comment:
can you also test actor_info_progress_str in different stages in this test case?